modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/manage_user_associated_data_applications_summary.jsp

```diff
@@ -14,4 +14,63 @@
  */
 --%>
 
-<%@ include file="/init.jsp" %>
\ No newline at end of file
+<%@ include file="/init.jsp" %>
+
+<%
+ManageUADApplicationsSummaryDisplay manageUADApplicationsSummaryDisplay = (ManageUADApplicationsSummaryDisplay)request.getAttribute(UserAssociatedDataWebKeys.MANAGE_UAD_APPLICATIONS_SUMMARY_DISPLAY);
+%>
+
+<div class="container-fluid container-fluid-max-xl container-form-lg">
+	<div class="sheet sheet-lg">
+		<div class="sheet-header">
+			<h2 class="sheet-title"><liferay-ui:message key="application-data-review" /></h2>
+		</div>
+
+		<div class="sheet-section">
+			<h3 class="sheet-subtitle"><liferay-ui:message key="status-summary" /></h3>
+			<strong><liferay-ui:message key="remaining-items" />: </strong><%= manageUADApplicationsSummaryDisplay.getTotalCount() %>
+		</div>
+
+		<div class="sheet-section">
+			<h3 class="sheet-subtitle"><liferay-ui:message key="applications" /></h3>
+
+			<liferay-ui:search-container
+				id="UADApplicationsSummaryDisplays"
+				searchContainer="<%= manageUADApplicationsSummaryDisplay.getSearchContainer() %>"
+			>
+				<liferay-ui:search-container-row
+					className="com.liferay.user.associated.data.web.internal.display.UADApplicationSummaryDisplay"
+					escapedModel="<%= true %>"
+					keyProperty="name"
+					modelVar="uadApplicationsSummaryDisplay"
```

Comment:
This one should be `uadApplicationSummaryDisplay` without the `s` for application